linux/debian/control

```diff
@@ -37,8 +37,8 @@ Build-Depends:
  metacity,
  gawk,
 Standards-Version: 4.6.1
-Vcs-Git: https://github.com/keymanapp/keyman.git [linux/debian]
-Vcs-Browser: https://github.com/keymanapp/keyman/tree/master/linux/debian
+Vcs-Git: https://github.com/keymanapp/keyman.git -b stable-16.0 [linux/debian]
+Vcs-Browser: https://github.com/keymanapp/keyman/tree/stable-16.0/linux/debian
```

Comment:
So will this have to change with every release?

Reply:
Yes. (That's one reason why I'd prefer if it'd be named `stable` and we would use tags to distinguish the different versions)

Reply:
We could consider this change for branch names -- but it would require a fair bit of work on tooling, so I'm not in a hurry to do it!

Reply:
Can you add instructions on this to the release wiki at https://github.com/keymanapp/keyman/wiki/Release-workflow?

Reply:
ok, done.